src/seedu/addressbook/data/person/UniquePersonList.java

```diff
@@ -110,6 +105,7 @@ public void add(Person toAdd) throws DuplicatePersonException {
         internalList.add(toAdd);
     }
 
+
```

Comment:
Plenty of random line breaks throughout. Would be good to omit this kind of changes, if possible.